cli/cliflags/names.go

```diff
@@ -33,6 +33,7 @@ const (
 	PasswordName          = "password"
 	PortName              = "port"
 	HTTPPortName          = "http-port"
+	HTTPIPAddrName        = "http-addr"
```

Comment:
small nit: use `HTTPAddrName`, it doesn't have to be an IP address, dns names work fine.